.github/workflows/ci.yaml

```diff
@@ -139,6 +126,47 @@ jobs:
           only-new-issues: true
           version: v1.42
 
+  chart-lint:
+    name: Helm chart Lint
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: helm/chart-testing-action@v2.1.0
+      - name: Set up helm-unittest
+        run: |
+          helm plugin install https://github.com/quintush/helm-unittest
+      - name: Lint charts
+        run: ct --config .github/ct.yaml lint --debug --all
+
+      - name: Run unit tests
+        run: |
+          for chart in $(ct --config .github/ct.yaml list-changed); do
```

Comment:
Simpler. I used the flag to see if they changed to run tests on all the charts that have tests.

Solved in [#791d18f](https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/40/commits/791d18f29f2c6f9c2237a4da1e47af02428be46b)

Reply:
I reverted it in [#db8a16c](https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/40/commits/db8a16c1308c7f9467a91b1a8b65f19365c452d6) because the pipeline could fail if there is a chart without tests.

Right now there is no chart without tests except when we first copy the chart to each chart to each repository but I would like to have the tests as consistent as I can.

Reply:
> the pipeline could fail if there is a chart without tests.
This actually could be a wanted behavior 😃 . But i'm fine with having this , only that if it's just to save some seconds on the executions I would favor simplicity/readability over performance.